pkg/virt-handler/vm.go

```diff
@@ -1462,6 +1537,25 @@ func (d *VirtualMachineController) processVmUpdate(origVMI *v1.VirtualMachineIns
 			}
 		}
 
+		// migrate old network cache files from older launcher pods
+		err := d.migrateNetworkCacheFiles(vmi)
```

Comment:
I love that here it looks like 3 tiny calls, but under the hood...

Good abstraction!